test/test_sparse.py

```diff
@@ -1906,6 +1901,101 @@ def test_asin_arcsin(self):
             )
             self._test_asin_arcsin(input_uncoalesced)
 
+    def _test_abs_absolute(self, sparse_tensor):
+        self._test_unary_op(
+            sparse_tensor, 
+            ops=(
+                torch.abs, torch.Tensor.abs,
+                torch.absolute, torch.Tensor.absolute,
+                operator.abs
+            ),
+            ops_out=(torch.abs, torch.absolute),
+            ops_inplace=(torch.Tensor.abs_, torch.Tensor.absolute_)
```

Comment:
Include `torch.abs_` to `ops_inplace`.

Perhaps OT: Is there a reason why `torch.absolute_` does not exist?